app/src/main/java/com/gatchimath/totask/Task.java

```diff
@@ -1,79 +1,90 @@
 package com.gatchimath.totask;
 
+import android.databinding.BaseObservable;
+import android.databinding.Bindable;
+
 /**
  * Basic task class.
  *
  * @author  gatchi (github.com/gatchi)
  */
-public class Task {
+public class Task extends BaseObservable {
 
 	private String name;
-	private String text;
+	private String details;
 	private boolean doneState;
 	
 	/**
 	 * @param name  The name and main content of this task.
-	 * @param text  Any additional details pertinent to the task.
+	 * @param details  Any additional details pertinent to the task.
 	 */
-	public Task(String name, String text) {
+	public Task(String name, String details) {
 		this.name = name;
-		this.text = text;
+		this.details = details;
 		this.doneState = false;
 	}
 	
 	/**
 	 * @return The name and main content of the task.
 	 */
+	@Bindable
 	public String getName() {
 		return name;
 	}
 	
 	/**
-	 * @param name  The name and main content of this task.
+	 * @return Any additional details pertinent to the task.
 	 */
-	public void setName(String name) {
-		this.name = name;
+	@Bindable
+	public String getDetails() {
+		return details;
 	}
 	
 	/**
-	 * @return Any additional details pertinent to the task.
+	 * @return True if the task is done, false if not.
 	 */
-	public String getText() {
-		return text;
+	@Bindable
+	public boolean getDoneState() {
```

Comment:
It does looks like data binding task.isDonw will automatically call the public method isDone() if it exists, so keep the old naming please.